### PR TITLE
Add `pyorbital` requirement of SEVIRI calibration in setup.

### PR DESCRIPTION
**Pull request by sbrodehl**
This PR adds missing extra requirements of the SEVIRI calibration algorithm, namely `pyorbital`, for the readers `seviri_l1b_hrit`, `seviri_l1b_native` and `seviri_l1b_nc` in [setup.py](setup.py).

I assumed the used version requirement is `>= 1.3.1` (similar to the `angles` / `hrpt` module).
Please let me know if this is not correct!

Without this package, loading channels in e.g. a `seviri_l1b_hrit`-scene fails (rather silently).

 - [x] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

**Comment by djhoese** [comment]
@ameraner @sfinkens Are you the last people to work on the SEVIRI readers? Should there maybe be a more obvious error/failure here rather than Satpy's typical silent "produce what we can"?

Otherwise, if either of you could review this that'd be great. I think this looks good. Thanks @sbrodehl.

File: setup.py [hunk 1]
```diff
@@ -56,6 +56,9 @@
     'nc_nwcsaf_msg': ['netCDF4 >= 1.1.8'],
     'sar_c': ['python-geotiepoints >= 1.1.7', 'rasterio', 'rioxarray'],
     'abi_l1b': ['h5netcdf'],
+    'seviri_l1b_hrit': ['pyorbital >= 1.3.1'],
+    'seviri_l1b_native': ['pyorbital >= 1.3.1'],
+    'seviri_l1b_nc': ['pyorbital >= 1.3.1'],
```
Comment (mraspaud) [inline setup.py:61]: That reader needs netCDF4 installed also.

**Review by mraspaud — COMMENTED** [review]
Thanks for the contribution! I Have one small comment otherwise it looks fine!

File: setup.py (hunk 1 above)
Comment (sbrodehl) [reply setup.py]: Great, I added netCDF4 as well!

**Review by mraspaud — APPROVED** [review]
LGTM

**Comment by mraspaud** [comment]
Merged, thanks a lot @sbrodehl !

**Comment by sfinkens** [comment]
Good catch, thanks for fixing this @sbrodehl !

**Pull request merged**